region/norfair/crocomire/Post Crocomire Shaft.json

```diff
@@ -660,7 +660,7 @@
       "requires": [
         "canTrickyUseFrozenEnemies",
         {"ammo": {"type": "Super", "count": 1}},
-        "h_useSpringBall",
+        "Morph",
```

Comment:
Its not obvious but you can jump onto the viola without morph if you aim down.  Otherwise you trip and touch the transition.